pr/webhook/pull_request.go

```diff
@@ -174,13 +174,13 @@ func (receiver *Receiver) processPullRequest(event *github.PullRequestEvent) {
 		if isSubmission {
 			typeLabels = appendIfUnique(typeLabels, "type: submission")
 		}
-		if strings.Contains(strings.ToLower(*event.PullRequest.Title), ": update to") {
+		if strings.Contains(strings.ToLower(*event.PullRequest.Title), ": update") || strings.HasPrefix(strings.ToLower(*event.PullRequest.Title), "update") {
```

Comment:
Please revert this part, checking for the word `update` may catch other non-updates like `neovim: bump revision, unibilium updated`.

Reply:
`: update` is acceptable though. Just remove the later part.

Reply:
Did you see that I check for update only at the beginning of the title? It would not catch your example above.
I can still remove it, I just want to make sure that you are awaqre that I check for update only with `HasPrefix`.